quickwit/quickwit-cli/src/split.rs

```diff
@@ -332,32 +335,24 @@ impl SplitCliCommand {
 }
 
 async fn list_split_cli(args: ListSplitArgs) -> anyhow::Result<()> {
-    debug!(args = ?args, "list-split");
-
-    let quickwit_config = load_quickwit_config(&args.config_uri).await?;
-    let metastore_uri_resolver = quickwit_metastore_uri_resolver();
-    let metastore = metastore_uri_resolver
-        .resolve(&quickwit_config.metastore_uri)
-        .await?;
-
-    let mut query = ListSplitsQuery::for_index(&args.index_id)
-        .with_split_states(args.split_states.unwrap_or_default());
-
-    if let Some(start_date) = args.start_date {
-        query = query.with_time_range_start_gte(start_date.unix_timestamp());
-    }
-    if let Some(end_date) = args.end_date {
-        query = query.with_time_range_end_lte(end_date.unix_timestamp());
-    }
-    if let Some(create_date) = args.create_date {
-        query = query.with_create_timestamp_lte(create_date.unix_timestamp());
-    }
-    if let Some(tags) = args.tags {
-        query = query.with_tags_filter(tags);
-    }
-
-    let splits = metastore.list_splits(query).await?;
-
+    debug!(args=?args, "list-split");
+    let transport = Transport::new(args.cluster_endpoint);
+    let qw_client = QuickwitClient::new(transport);
+    let list_splits_query_params = ListSplitsQueryParams {
+        split_states: args.split_states,
+        start_timestamp: args.start_date.map(OffsetDateTime::unix_timestamp),
+        end_timestamp: args.end_date.map(OffsetDateTime::unix_timestamp),
+        end_create_timestamp: args.create_date.map(OffsetDateTime::unix_timestamp),
+    };
+    // TODO: plug tags.
+    // if let Some(tags) = args.tags {
```

Comment:
maybe we can create the ticket and reference it here

Reply:
will do